csa-runner-app/src/main/java/com/farao_community/farao/swe_csa/app/dichotomy/DichotomyRunner.java

```diff
@@ -113,6 +113,14 @@ public RaoResult runDichotomy(CsaRequest csaRequest) throws GlskLimitationExcept
             double ctPtEsMax = expPtEs0 >= 0 ? Math.min(Math.min(-ctRaPtEs.getMinAdmissibleSetpoint(expPtEs0), ctRaEsPt.getMaxAdmissibleSetpoint(expEsPt0)), expPtEs0)
                 : Math.min(Math.min(ctRaPtEs.getMaxAdmissibleSetpoint(expPtEs0), -ctRaEsPt.getMinAdmissibleSetpoint(expEsPt0)), -expPtEs0);
 
+            if (ctFrEsMax < Math.abs(expFrEs0)) {
```

Comment:
actually this doesn't work, because it's already the min between the RA's range and the initial exchange (see line 111)

Reply:
I'll revert and add some unit test to ensure that every case is OK